src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindowFactory.kt

```diff
@@ -17,6 +20,14 @@ class TestGenieToolWindowFactory : ToolWindowFactory {
     override fun createToolWindowContent(project: Project, toolWindow: ToolWindow) {
         val testGeniePanelWrapper = TestGenieToolWindow()
         val contentFactory : ContentFactory = ContentFactory.SERVICE.getInstance()
+
+        val testCaseDisplayService = project.service<TestCaseDisplayService>()
+        toolWindow.contentManager.addContent(
+            contentFactory.createContent(testCaseDisplayService.mainPanel,
```

Comment:
This looks a bit inconsistent with adding content below. Maybe you can change the indentation here to be more consistent with the one below?

Reply:
Oops, I may have forgotten to format?